iree/compiler/Dialect/Util/IR/UtilOpFolders.cpp

```diff
@@ -308,6 +308,79 @@ void RangeExtentsOp::getCanonicalizationPatterns(
   results.insert<DeduplicateRangeExtentsOp>(context);
 }
 
+//===----------------------------------------------------------------------===//
+// util.align
+//===----------------------------------------------------------------------===//
+
+// TODO(#5405): add canonicalizers that reach further in the IR or a dedicated
+// pass for full potential-value-set analysis.
```

Comment:
just dropping an issue tag for history: https://github.com/google/iree/issues/5405

also cc @KoolJBlack